vector-im/element-android
# [Poll] Warning message on decryption failure of some events (PSG-1025)

## Type of change


## Content


Making the encrypted events as processable. Processing the events with `relatesTo` field. When an event relates to an existing poll, it is saved into the related summary entity so that a warning can be displayed to the user.

## Motivation and context


Closes #7824 

## Screenshots / GIFs





<img src="https://user-images.githubusercontent.com/46314705/209679559-e573fee6-7fba-4685-9919-21408695beda.png" width=300 />


## Tests


The decryption failure is rather hard to test. The UI has been tested by mocking decryption failure.
We should ensure poll is working as before.

- Create open or closed poll in a room
- Vote on the poll
- Check the UI for the message is correct
- End the poll
- Check the UI for the message is correct

## Tested devices

- OS version(s): Android 11

## Checklist
